command/sub8_launch/launch/subsystems/cameras/calibration/00b09d0100ea51be.yaml

```diff
@@ -1,6 +1,6 @@
 image_width: 644
 image_height: 482
-camera_name: 00b09d0100ea51be
+camera_name: 15356350
```

Comment:
Have you verified that these do not change on reboot? Keep in mind that these might be tied to the OS if they changed because of the reinstall, and will thus change again when we switch to our permanent OS.